### ISSUE TITLE
node 14.13 import problems (assert_1.default.ok is not a function)

### ISSUE THREAD

**Issue by arlac77** [benjamn/recast#776]
Starting with node 14.13.0 I got the following error:

```
TypeError: assert_1.default.ok is not a function

  › fromString (node_modules/recast/lib/lines.js:580:22)
  › Object.<anonymous> (node_modules/recast/lib/lines.js:654:18)
  › Object.<anonymous> (node_modules/recast/lib/parser.js:11:15)
```

**Comment by eventualbuddha** [comment]
Can you post some steps to reproduce this error? I tried loading recast and doing some basic stuff in node 14.13.0 with recast and it seems fine.

**Comment by arlac77** [comment]
Sure, my 1st. try to  narrow down the problem did not work.
Simply calling recast does not reproduce the problem.
Somehow it depends on other imported modules (or the order of them).
[Travis log](https://travis-ci.com/github/arlac77/npm-template-sync/jobs/392937762)

**Comment by arlac77** [comment]
Found a combination of imports that does not work any longer with node 14.13.0
```js
import recast from "recast";
import lockfile from '@yarnpkg/lockfile';
```

**Comment by arlac77** [comment]
Fixed with node 14.13.1

**Comment by jasonlbw** [comment]
i still have this problem with node 14.19.3

**Comment by gnprice** [comment]
@jasonlbw Probably best to file a new issue. Please include steps to reproduce the problem.

The original issue here turned out to be a Node bug, and it was fixed in Node 14.13.1. So whatever issue you're seeing likely has a different cause than this one, even if it has similar symptoms.
